README.md

```diff
@@ -1,26 +1,99 @@
-# tilegarden [![Build Status](https://travis-ci.org/azavea/tilegarden.svg?branch=develop)](https://travis-ci.org/azavea/tilegarden)
+# tilegarden :world_map::sunflower: [![Build Status](https://travis-ci.org/azavea/tilegarden.svg?branch=develop)](https://travis-ci.org/azavea/tilegarden)
+
+## Contents
+ * [About](#About)
+ * [Usage](#Usage)
+   * [Local Development](#local-development)
+     * [Map Styles](#map-styles)
+     * [Filters](#filters)
+     * [UTF Grids](#utf-grids)
+     * [Vector Tiles](#vector-tiles)
+     * [Debugging](#debugging)
+   * [Deployment to AWS](#deployment-to-aws)
+     * [Additional Configuration Options](#additional-configuration-options)
+     * [Required AWS Permissions](#required-aws-permissions)
+ * [Helpful Links](#helpful-links)
+
+## About
+Tilegarden is a serverless tile-rendering tool using [Mapnik](http://mapnik.org/), built for AWS Lambda. Serve custom-generated map tiles without having to worry about server maintenance, scaling, or paying for resources that aren't being accessed.
```

Comment:
Maybe mention PostGIS here, too.  The expectations built in around what the datasource is seem like information on the same level as this--helpful to know up front so you can understand the nature of the thing.

Reply:
Are there actually expectations about the datasource? I'm pretty it'll support any datasource Mapnik does, I've tested it on local shapefiles and it's worked fine.

Reply:
Although now that I think about it, to use something like local shapefiles you would probably have to store them in S3, and grabbing stuff from S3 isn't something we've implemented yet. Also, tiling and UTF grids sort of hinge on PostGIS being the datasource, so maybe I _should_ add that in.